data/reusables/actions/jobs/setting-default-values-for-jobs-defaults-run-working-directory.md

```diff
@@ -0,0 +1,10 @@
+Use `working-directory` to define the working directory for the `shell` for a step. {% data reusables.actions.reusable-keyword %}
+
+{% data reusables.actions.working-directory-must-exist %}
+
+The `working-directory` for a step is taken from the first defined in this list:
+1. The step `jobs.<job_id>.steps[*].run.working-directory`. See [`jobs.<job_id>.steps[*].working-directory`](/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory).
+1. The job `jobs.<job_id>.defaults.run.working-directory`. See [`jobs.<job_id>.defaults.run.working-directory`](/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iddefaultsrunworking-directory).
+1. The workflow `defaults.run.working-directory`. See [`defaults.run.working-directory`](/actions/using-workflows/workflow-syntax-for-github-actions#defaultsrunworking-directory).
```

Comment:
```suggestion
```
Similar to above, I don't think we should explicitly define this process in the docs.